src/modules/handlers/hello.ts

```diff
@@ -0,0 +1,7 @@
+async function hello (ctx) {
```

Comment:
Also, this entire file is a router, so how about create a `routers` folder for this ?